spec/spec_helper.rb

```diff
@@ -1,13 +1,13 @@
 # Set up coverage analysis
 #-----------------------------------------------------------------------------#
 
-if RUBY_VERSION >= '1.9.3'
-  require 'codeclimate-test-reporter'
-  CodeClimate::TestReporter.configure do |config|
-    config.logger.level = Logger::WARN
-  end
-  CodeClimate::TestReporter.start
-end
+# if RUBY_VERSION >= '1.9.3'
```

Comment:
@amorde should this be commented out?

Reply:
Yeah so the current version of rubicop is broken on this ruby and updating it triggers a whole bunch of rubocop errors from outdated configs. Intended to come back to it